README.md

```diff
@@ -58,7 +59,7 @@ When possible, use the `withIsMobile` and `useIsMobile` for mobile devices detec
 
 | Key                    | Type    | Description                                                                                  |
 |------------------------|---------|----------------------------------------------------------------------------------------------|
-| isMobile | boolean | If its below the md breakpoint |
+| isMobile | boolean | if its below the mobile breackpoint defined by mobileBreakpoint |
```

Comment:
if its -> If it's
breackpoint -> breakpoint

Reply:
Done thanks 👍